Marlin/src/gcode/feature/trinamic/M569.cpp

```diff
@@ -24,10 +24,6 @@
 
 #if HAS_STEALTHCHOP
 
-#if AXIS_COLLISION('I')
-  #error "M569 parameter collision with axis name."
-#endif
-
```

Comment:
It would be good to preserve G-code standards whenever possible in spite of potential axis collisions, rather than impose on users that axes are sometimes referred to by their assigned name and sometimes by their "private" name. It would be best to conceal "IJKUVW" from users as much as possible and let users figure out on their own how to deal with situations where standard parameter names conflict with their desired axis names.

Reply:
I agree that using AXIS_COLLISION('B') and similar tests for valid axis names (ABCUVW) is a good approach. However we do not allow AXIS*_NAME other than ABCUVW per sanitycheck.h because that would break sooo many things. Thus, AXIS_COLLISION('I') is redundant, IMHO.